imap_upstream.adoc

```diff
@@ -136,7 +136,7 @@ behavior:
 complete. This will cause all existing IMAP sessions to be restarted
 on the new servers.
 2. One day prior to switching to the imapd pool, modify the value
-`zimbraMemcachedClientExpirySeconds` of to a shorter interval, such as
+`zimbraReverseProxyCacheEntryTTL` of to a shorter interval, such as
```

Comment:
The word "of" should come before the parameter being modified.

Reply:
I'll modify this, too!